src/styles/form.css

```
@@ -0,0 +1,66 @@
+.search-form > * {
+  margin-top: 20px;
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  padding: 24px 25px 25px;
+}
+
+.search-bar:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__img {
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  width: 19px;
+  height: 19px;
+  flex-shrink: 0;
+}
+
+.search-bar__input {
+  width: 100%;
+  border: none;
+  outline: none;
+  padding: 0;
+  padding-left: 18px;
+  height: 19px;
+  font-family: Avenir-Book, sans-serif;
```

apply font-family for `html` tag selector and let inputs inherit it

```suggestion
  font-family: inherit;
```